.github/workflows/go.yml

```diff
@@ -73,8 +73,8 @@ jobs:
     # Dependency for Go module github.com/google/gopacket
     - name: Install libpcap-dev
       run: sudo apt-get -y install libpcap-dev
-    - name: Go vet
-      run: GOGC=30 go vet ./...
+#    - name: Go vet
```

Comment:
uncomment `go vet`?

Reply:
I had commented it to get some results faster - reenabled now.